tensorflow/python/data/experimental/benchmarks/snapshot_dataset_benchmark.py

```diff
@@ -42,93 +41,143 @@ def _makeSnapshotDirectory(self):
     os.mkdir(tmp_dir)
     return tmp_dir
 
-  def _createSimpleDataset(self, num_elems, tmp_dir=None,
+  def _createSimpleDataset(self, num_elements, tmp_dir=None,
                            compression=snapshot.COMPRESSION_NONE):
     if not tmp_dir:
       tmp_dir = self._makeSnapshotDirectory()
 
     dataset = dataset_ops.Dataset.from_tensor_slices([1.0])
     dataset = dataset.map(
         lambda x: gen_array_ops.broadcast_to(x, [50, 50, 3]))
-    dataset = dataset.repeat(num_elems)
+    dataset = dataset.repeat(num_elements)
     dataset = dataset.apply(
         snapshot.legacy_snapshot(tmp_dir, compression=compression))
 
     return dataset
 
-  def _consumeDataset(self, dataset, num_elems):
-    dataset = dataset.skip(num_elems)
-    next_element = dataset_ops.make_one_shot_iterator(dataset).get_next()
-    with session.Session() as sess:
+  def _consumeDataset(self, dataset, num_elements):
```

Comment:
this method should be moved to the benchmark base

Reply:
addressed in the commit.